Changelog.rst

```diff
@@ -112,6 +112,38 @@ For instance, ``iptables::tcp_stateful_listen`` is now ``iptables::listen::tcp_s
 Additionally, any ``add_rule`` defines were changed to just ``rule``. For
 example, ``auditd::add_rule`` was changed to just ``auditd::rule``.
 
+Centralized Management of Application x509 PKI Certs
```

Comment:
👍 Added a ticket to add more like this to the real docs: https://simp-project.atlassian.net/browse/SIMP-2544